### PR TITLE
Add daily-check.md

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

Add daily-check.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3221
- Other reference link(s):

### CODE REVIEW

File: daily-check.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
++ **Status**: This indicator is used to check whether the status is normal. For an online node, this can be ignored.
++ **Up Time**: The key indicator. If you find that the `Up Time` is changed, you need to locate the reason why the component is restarted.
++ **Version**, **Deployment Directory**, **Git Hash**: These indicators need to be checked to avoid inconsistent or even incorrect version/deployment directory.
+
+### Host panel
+
+![Host panel](/media/host-panel.png)
+
+You can view the usage of CPU, memory, and disk. When the average usage of any resource exceeds 60%, it is recommended to plan to scale out the capacity. When the average usage reaches 80%, it is recommended to scale out the capacity.
```
Comment (WalterWj) [inline daily-check.md:29]: 通过主机面板可以查看 CPU、内存、磁盘使用率。当任何资源的平均使用率超过 60%，推荐安排扩容计划，达到 80%，推荐进行扩容。 这个原文写错了，改成：
通过主机面板可以查看 CPU、内存、磁盘使用率。当任何资源的平均使用率超过 80%，推荐安排扩容计划。

File: daily-check.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+### SQL analysis panel
+
+![SQL analysis panel](/media/sql-analysis-panel.png)
+
+You can locate the slow SQL statement executed in the cluster. Then you can optimize the specific SQL statement.
+
```
Comment (ran-huang) [inline daily-check.md:36]:
```suggestion

## Monitoring metrics

```

File: daily-check.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+### Region panel
+
+![Region panel](/media/region-panel.png)
+
++ `miss-peer-region-count`: The number of Regions without enough replicas. This value is not always greater than `0`.
++ `extra-peer-region-count`: The number of Regions with extra replicas. These Regions are generated during the scheduling process.
++ `empty-region-count`: The number of empty Regions, generated by executing the `TRUNCATE TABLE`/`DROP TABLE` statement. It this number is large, you can consider enabling `Region Merge` to merge Regions across tables.
```
Comment (ran-huang) [inline daily-check.md:43]:
```suggestion
+ `empty-region-count`: The number of empty Regions, generated by executing the `TRUNCATE TABLE`/`DROP TABLE` statement. If this number is large, you can consider enabling `Region Merge` to merge Regions across tables.
```

File: daily-check.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
++ `pending-peer-region-count`: The number of Regions with outdated Raft logs. It is normal that a few pending peers are generated in the scheduling process. However, it is not normal if this value is large for a period of time.
++ `down-peer-region-count`: The number of Regions with an unresponsive peer reported by the Raft leader.
++ `offline-peer-region-count`: The number of Regions when removing peers.
```
Comment (ran-huang) [inline daily-check.md:46]:
```suggestion
+ `offline-peer-region-count`: The number of Regions during the offline process.
```

File: daily-check.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Generally, it is normal that these values are not `0`. However, it is not normal that they are not `0` for quite a long time.
+
```
Comment (ran-huang) [inline daily-check.md:49]:
```suggestion

## Response time

```

File: daily-check.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+### KV Request Duration
+
+![TiKV request duration](/media/kv-duration-panel.png)
+
+The KV request duration 99 in TiKV. If you find nodes with a long duration, you can check whether there are hot spots, or whether there are nodes with poor performance.
```
Comment (ran-huang) [inline daily-check.md:54]:
```suggestion
The KV request duration 99 in TiKV. If you find nodes with a long duration, check whether there are hot spots, or whether there are nodes with poor performance.
```

File: daily-check.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+### PD TSO Wait Duration
+
+![TiDB TSO Wait Duration](/media/pd-duration-panel.png)
+
+The time it takes TiDB to obtain TSO from PD. The following lists reasons for long duration:
```
Comment (ran-huang) [inline daily-check.md:60]:
```suggestion
The time it takes for TiDB to obtain TSO from PD. The following are reasons for the long wait duration:
```

File: daily-check.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
++ High network latency from TiDB to PD. You can manually execute the ping command to test the network latency.
++ High load for the TiDB server.
++ High load for the PD server.
+
```
Comment (ran-huang) [inline daily-check.md:65]:
```suggestion

## Hardware monitoring

```

File: daily-check.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+### Overview panel
+
+![Overview panel](/media/overview-panel.png)
+
+You can view the load, memory available, network traffic, and IO utilities. When a bottleneck is found, it is recommended to scale out the capacity, or to optimize the cluster topology, SQL, cluster parameters, etc.
```
Comment (ran-huang) [inline daily-check.md:70]:
```suggestion
You can view the load, memory available, network traffic, and I/O utilities. When a bottleneck is found, it is recommended to scale out the capacity, or to optimize the cluster topology, SQL, cluster parameters, etc.
```

File: daily-check.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+### Exceptions
```
Comment (ran-huang) [inline daily-check.md:72]:
```suggestion
## Exceptions
```

File: daily-check.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+
+![Exceptions](/media/failed-query-panel.png)
+
+You can view the errors triggered by the execution of SQL statements on each TiDB instance. These include syntax error, primary key conflicts, etc.
+
+### GC status
```
Comment (ran-huang) [inline daily-check.md:78]:
```suggestion
## GC status
```

**Comment by ran-huang** [comment]
@WalterWj Please sign the CLA so we can merge this PR~ Thanks!

File: daily-check.md (hunk 1 above)
Comment (toutdesuite) [reply daily-check.md:29]: updated

**Pull request merged**